griptape/common/prompt_stack/contents/action_call_delta_message_content.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import annotations
+from attrs import define, field
+from typing import Optional
+
+from griptape.common import BaseDeltaMessageContent
+
+
+@define
+class ActionCallDeltaMessageContent(BaseDeltaMessageContent):
```

Comment:
Same with all events...I don't like the long names but we should probably stay consistent.